### PR TITLE
updated t(STRING) instances to requested headers

### LINKED ISSUE #3932: Remove Translations 16: update t() in app/views/learning_hours/_update_form.html.erb
Thank you for coming to our workshop!

To complete this issue:
1. open `app/views/learning_hours/_update_form.html.erb`
1. find each call of `t(STRING)`
1. look up `STRING` in `config/locales/views.en.yml`
    1. the YAML file is nested, first with the language (always `en` here), then the view name, then the action name
    1. so for your `STRING`, look under `en:` then `learning_hours:` then `some_action_name:` and then `STRING:`
    1. if you need any help figuring the view or action name to look up the translation with, please ask a workshop helper
1. replace the usage of `t(` with the translation you found in `config/locales/views.en.yml`

This will simplify our templates and make it easier to maintain the code going forward.

Example:

```
Before:
<span><%= t(".title") %></span>

In config/locales/views.en.yml:
en:
  learning_hours:
    some_action_name:
      title: Hello world

After:
<span>Hello world</span>
```

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by katmlane**
### What github issue is this PR for, if any?
Resolves #3932

### What changed, and why?
Updated t(STRING) instances with the proper replacement found in the views.en.yml file. This will help manage the code better going forward.

### Feelings gif (optional)
 
`alt text`

### CODE REVIEW

**Comment by katmlane** [comment]
@compwron  please let me know what I can do to improve My contribution process!

**Comment by katmlane** [comment]
What github issue is this PR for, if any?
Resolves https://github.com/rubyforgood/casa/issues/3931

What changed, and why?
Updated t(STRING) instances with the proper replacement found in the views.en.yml file. This will help manage the code better going forward.

Feelings gif (optional)
alt text

File: app/views/learning_hours/_update_form.html.erb [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-    <%= form.label :name, t(".learning_hours_title") %>
+    <%= form.label :name, Learning Hours Title %>
     <%= form.text_field :name,
                         id: "title",
                         placeholder: "-- Enter a title --",
                         value: @learning_hour.name,
                         class:"mr-5" %>
     <br>
-    <%= form.label :learning_type, t(".learning_type") %>
+    <%= form.label :learning_type, Type of Learning %>
```
Comment (littleforest) [inline app/views/learning_hours/_update_form.html.erb:18]:
```suggestion
    <%= form.label :learning_type, "Type of Learning" %>
```

**Comment by littleforest** [comment]
@compwron or @katmlane can you merge or rebase in main? The broken docker spec is fixed there.

**Comment by katmlane** [comment]
@littleforest  @compwron  I see now that I was missing the quotations around the title replacements. This was my first attempt at open source contribution and I don't feel comfortable rebasing. I've attempted to merge in main and it says it's up to date.

**Comment by thoughtbot-boost-support** [comment]
> I've attempted to merge in main and it says it's up to date.

You are working on a fork, so it is a little confusing to pull in changes from the main casa repo. I just went ahead and updated your branch with the latest changes from the main casa repo.

**Comment by littleforest** [comment]
>  I've attempted to merge in main and it says it's up to date.

You are working on a fork, so it is a little confusing to pull in changes from the main casa repo. I just went ahead and updated your branch with the latest changes from the main casa repo.

**Review by compwron — APPROVED** [review]
It looks good now, let's merge! :)

**Pull request merged**